chall03/ssacrist.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import sys
+
+def byebye():
+    print("usage: %s <1-9 squared_rows...>" % sys.argv[0])
+    sys.exit()
+
+def check():
+    if len(sys.argv) < 2 or len(sys.argv) > 21:
+        byebye()
+    sides = (len(sys.argv)) - 1
+    long = (len(sys.argv[1]))
+    if sides != long:
+        byebye()
+    for i in range(2, sides+1):
+        if len(sys.argv[i]) != long or (sys.argv[i]).isdigit() == False:
```

Comment:
Comparison to False should be 'if cond is False:' or 'if not cond:'

Reply:
I don't understand at all. Before uodate the, I type  `-ls -la` the permission, and I check the permission: -rwxrwxrwx.  Is it necesary to do any other action??

Reply:
Maybe you were in another branch?

You are submitting your PR from your fork:master-branch. And when I clone your repo and I check the master branch, the permissions in your file are `-rw-r--r--` (644) :(